src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc"),
+    url("./fonts/Avenir-Heavy.ttf"),
+    url("./fonts/Avenir-Book.ttf");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.searchform {
+  display: block;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.boxes {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.bigsearch {
+  width: 100%;
+  padding: 23px 10px 23px 62px;
+  background: #fff;
+
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.smallsearch {
+  width: 100%;
+  height: 42px;
+  padding: 12px 12px 10px 33px;
+
+  font-family: inherit;
+  font-size: 14px;
+  line-height: 19px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 13px 15px;
+  background-size: 11px 11px;
+}
```

Comment:
Don't repeat yourself. Properties background-image, background-repeat, font-family, and width are common for both bars. Move it to common class